commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java

```diff
@@ -40,6 +42,14 @@ public static void setup() {
 
   }
 
+  @AfterEach
+  public void signout() {
```

Comment:
I would move this method under openPlatform method to make it more readable (before/after).

Reply:
done